pkg/api/short_url.go

```diff
@@ -15,12 +15,12 @@ import (
 
 // createShortURL handles requests to create short URLs.
 func (hs *HTTPServer) createShortURL(c *models.ReqContext, cmd dtos.CreateShortURLCmd) response.Response {
-	hs.log.Debug("Received request to create short URL", "path", cmd.Path)
+	c.Logger.Debug("Received request to create short URL", "path", cmd.Path)
```

Comment:
```suggestion
	hs.log.Debug("Received request to create short URL", "path", cmd.Path)
```

Reply:
I had replaced hs.log with c.logger. Otherwise I would have to set log which is private value.